README.md

```diff
@@ -23,6 +23,13 @@ then please ignore these instructions.
 
 ## Adding a node
 
+### Minimum Hardware Requirements
+
+The minimum hardware requirements for running a node are:
+- 10GB Disk
+- 2GB RAM
```

Comment:
Arbitrary but I don't think we would want to encourage anything lower.

I was thinking maybe the installer should explicitly check this and fail to install if not met. If that makes sense I can do in a follow-up